added Tools.InnoSetup.props for msbuild support

TiltonJH opened the pull request:
added Tools.InnoSetup.props; makes the Inno Setup nuget tools directory available via $(InnoSetupDir) during msbuild

TiltonJH committed:
added Tools.InnoSetup.props; makes the Inno Setup nuget tools directory available via $(InnoSetupDir) during msbuild

vslavik commented:
Can you please provide a PR comment or better yet, [conventions-respecting](https://chris.beams.io/posts/git-commit/) git commit message explaining the question "Why?"

It isn't clear to me both what this PR does and what it's useful for.

TiltonJH commented:
NuGet will install the `Tools.InnoSetup.props` into the project which references the Tools.InnoSetup NuGet package. Which in turn provides the property `$(InnoSetupDir)`.

`$(InnoSetupDir)` point to the tools directory of the installed package. Now one can use something like `<Exec Command="$(InnoSetupDir)ISCC.exe someSetup.iss"/>` in MSBuild.

vslavik commented:
…replacing the need to include version number in the path to ISCC. OK, that makes sense, thanks!

vslavik commented:
Wouldn't it make even *more* sense to have, say, `$(InnoSetupCompiler)` pointing directly to the one executable that matters, though?

TiltonJH commented:
Sure, would just be another entry in `Tools.InnoSetup.props`.

TiltonJH commented:
Shall go and add this?

vslavik commented:
If you can, thank would be great!

TiltonJH committed:
.nuspec extended with .props for MSBuild integration.

NuGet will install the Tools.InnoSetup.props into the project which references the Tools.InnoSetup NuGet package. Which will in turn provide the properties $(InnoSetupDir) and $(InnoSetupCompiler).

$(InnoSetupDir) points to the tools directory of the installed package and $(InnoSetupCompiler) points directly at the ISCC.exe. Now one can use something like <Exec Command="$(InnoSetupDir)ISCC.exe someSetup.iss"/> or even simpler <Exec Command="$(InnoSetupCompiler) someSetup.iss"/> in MSBuild.

TiltonJH commented:
Will add a new pull request with the extension of $(InnoSetupCompiler).
